distributions/distribution-resources/src/main/resources/conf/sitemaps/readme.txt

```diff
@@ -0,0 +1,7 @@
+
+openHAB2 conf/sitemaps folder
+
+Your sitemap definitions go here.
+All sitemap files have to have the ".sitemap" file extension and must follow a special syntax.
+
+Check out the openHAB Wiki for more details: https://github.com/openhab/openhab/wiki/Explanation-of-Sitemaps
```

Comment:
Again, http://docs.openhab.org/features/sitemap.html should be filled with the wiki content.